examples/references/segmentation/pascal_voc2012/experiments/mlflow/conda.yaml

```diff
@@ -6,8 +6,8 @@ dependencies:
   - python=3.7
   - numpy
   - cudatoolkit=10.1.243=h6bb024c_0
-  - pytorch=1.3.1=py3.7_cuda10.1.243_cudnn7.6.3_0
-  - torchvision=0.4.2=py37_cu101
+  - pytorch=1.5.0=py3.7_cuda10.1.243_cudnn7.6.3_0  
```

Comment:
Does pin a specific build related to `amp` ? Or to ensure that `cuda` version is installed ? I don’t know if it’s a good idea to impose `cuda` version.

Reply:
Good point about amp version and pytorch. The reason of fixing versions here is more or less historical and was related to BC incompatibilities between torchvision and pillow...